docs/introduction/architecture.md

```diff
@@ -3,51 +3,46 @@ title: Shard Architecture
 
 ## Treasury Account
 
-In the genesis of the network we also specify a treasury account, which holds all the tokens left after the distribution of tokens to validators specified at creation. These tokens can be owned by a multisig of multiple parties or by one, depending on the network use case. This gives us the ability to create new accounts using the treasury account, deploy contracts, but also onboard new validators if needed. The private shard can be started by a single entity and using this approach onboard new validators into the network eventually.
+In the genesis of the network, a treasury account is specified to hold all the tokens remaining after the distribution of tokens to the validators specified during creation. The ownership of these tokens can be assigned to a multisig account involving multiple parties or a single entity, depending on the network's use case. The treasury account enables the creation of new accounts, contract deployment, and onboarding of new validators if necessary. A private shard can be initiated by a single entity, and this approach allows for the subsequent onboarding of new validators into the network.
 
 ## Account model
 
-We are integrated with the NEAR wallet, which means that users are using their NEAR Mainnet or Testnet FullAccess Key to authenticate with the Calimero Shard. Users sign an authorization message with the FullAccess Key asking for permission to the Authorization Service for a specific shard and if they are able to interact they will receive a JWT token for communicating with the Private Shard. This ensures that only the user with the corresponding key is actually issuing transactions on the shard.
-On top of that, Private shards use the NEAR account model inside the shard and will generate FunctionCall keys for specific contract interaction improving on security for the users and network.
+Calimero Shard is integrated with the NEAR wallet, which means that users authenticate using their NEAR Mainnet or Testnet FullAccess Key. Users sign an authorization message with the FullAccess Key to request permission from the Authorization Service for a specific shard. If they are granted permission, they receive a JWT token for communicating with the Private Shard. This authentication process ensures that only users with the corresponding key can issue transactions on the shard. Additionally, Private shards employ the NEAR account model within the shard and generate FunctionCall keys for specific contract interactions, enhancing security for users and the network.
```

Comment:
Additionally, private shards employ the NEAR account model within the shard and generate FunctionCall keys for specific contract interactions, enhancing security for users and the network.


Can we maybe link to https://docs.near.org/concepts/basics/accounts/access-keys#function-call-keys

or write a paragraph ourselves